packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsInfo/PayableDetailsInfo.tsx

```diff
@@ -154,7 +175,16 @@ const PayableDetailsInfoBase = ({
             <Table>
               <TableBody>
                 <TableRow>
-                  <StyledLabelTableCell>
+                  <StyledLabelTableCell
+                    style={{
+                      color: isFieldRequired(
+                        'invoiceNumber',
+                        payable.document_id
+                      )
+                        ? theme.palette.error.main
```

Comment:
A couple of comments here:
1. We're coupling our code with a child component's implementation here.
2. This code repeats itself in several places.

We can deduplicate the code by creating and new class like

    interface StyledLabelTableCellProps {
      children?: React.ReactNode;
      isError?: boolean;
    }
    const StyledLabelTableCell = (props: StyledLabelTableCellProps) => {
      const theme = useTheme();
      return <TableCell sx={{
        color: props.isError ? theme.palette.error.main : theme.palette.text.secondary,
        minWidth: 120,
        width: '35%', 
      }}>{props.children}</TableCell>;
    };